industrial_robot_simulator/CMakeLists.txt

```diff
@@ -2,11 +2,11 @@ cmake_minimum_required(VERSION 2.8.12)
 
 project(industrial_robot_simulator)
 
-find_package(catkin REQUIRED COMPONENTS std_msgs sensor_msgs
-  control_msgs trajectory_msgs)
+find_package(catkin REQUIRED COMPONENTS std_msgs sensor_msgs 
+  control_msgs trajectory_msgs industrial_msgs)
```

Comment:
This is a pure Python package, so the best thing to do would probably be to remove all `COMPONENTS` from the `find_package(..)` call, instead of adding `industrial_msgs`.

Then make sure all msg packages are listed as `exec_depend` in the manifest.